src/app/features/container/utils/get-title-from-url.ts

```diff
@@ -1,36 +0,0 @@
-import { RouteUrls } from '@shared/route-urls';
-
-export function getTitleFromUrl(pathname: RouteUrls) {
-  if (pathname.match(RouteUrls.SendCryptoAsset)) {
-    // don't show send on first step of send flow or popuop transfer
-    if (pathname === RouteUrls.SendCryptoAsset) return undefined;
-    if (pathname === RouteUrls.RpcSendTransfer) return undefined;
-    return 'Send';
-  }
-
```

Comment:
Previously I was using this pathname based lookup to indicate the title to be shown